### PR TITLE
feat: fixed formatiing and fixed layout

### PR DESCRIPTION
**Pull request by johnnyBravo8912**

### CODE REVIEW

File: frontend/src/modules/home/views/Home.vue [hunk 1]
```diff
@@ -29,13 +29,10 @@ const changeLanguage = () => {
...
-      <div class="format inner-container custom">
-        <div class="block-spacer h-1/3"></div>
+    <div class="base-container flex flex-col gap-4 justify-end items-center">
+      <div class="format inner-container">
         <div class="content flex justify-between">
-          <div class=" flex flex-col flex-1  gap-4">
+          <div class="flex flex-col flex-1 gap-4">
             <span class="text-custom-primary">{{
               langTranslations.aboutMeHeader
             }}</span>
```
Comment (sain0136) [inline frontend/src/modules/home/views/Home.vue:38]: Nice Job with everything overall John I am impressed by what you did here. But fill in this text then I will merge but excellent job on Css

**Review by sain0136 — CHANGES_REQUESTED** [review]

File: frontend/src/modules/home/views/Home.vue (hunk 1 above)
Comment (johnnyBravo8912) [reply frontend/src/modules/home/views/Home.vue:38]: K on it

Comment (sain0136) [reply frontend/src/modules/home/views/Home.vue:38]: Nice

**Review by sain0136 — APPROVED** [review]
Well done

**Pull request merged**